pkg/blaker/blaker.go

```diff
@@ -40,12 +41,17 @@ type RunCmdInput struct {
 }
 
 func (b *Blaker) RunCmd(input *RunCmdInput) (cmd.Status, error) {
-	breakTime, err := b.getBreakTime()
+	err := b.CanRunCmd(b.clock.Now())
```

Comment:
Instead of returning an error message, perhaps a method like IsCmdRunnable could be a better approch. 
If we have that one, we can both export the method as blaker API and usable in CLI.